styles.css

```diff
@@ -347,6 +347,17 @@ color: white !important;
     color: [[setting:newavatarbadgehovertextcolour]] !important;
 }
 
+.hsuforum-unreadcount {
+    background-color: [[setting:newavatarbadgebackgroundcolour]] !important;
+    color: [[setting:newavatarbadgetextcolour]] !important;
+    border-radius: [[setting:newavatarbadgeborderradius]];
+    border-color: [[setting:newavatarbadgebordercolour]];
```

Comment:
Was this working on your local. In getsmarter I do not thing we handle settings this way anymore.

Reply:
yes the button also working on prep310 staging, i just forgot to add for the post new link

Reply:
👍